### PR TITLE
remove obselete junk

### PR DESCRIPTION
**Pull request by MaxAbrahams**
pulll me

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -3,10 +3,9 @@
 <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-<script src="https://d3js.org/d3.v4.min.js"></script>
 <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
 <script type='text/javascript'>
-	//js code  here
+	//load google charts and define a callback for button press
```
Comment (wynged) [inline home.html]: good commenting

Comment (wynged) [inline home.html]: good commenting, like what you did.

File: home.html [hunk 2]
```diff
@@ -58,9 +57,7 @@
 <input type="text" class="form-control" id="client_id">
 </div>
 <button type="button" class="btn" onclick="graphWaveform">FetchAndDraw</button>
-<p>A d3 vector!!!!</p>
```
Comment (wynged) [inline home.html]: as a commenter, I also agree with this.

File: home.html [hunk 3]
```diff
@@ -3,10 +3,9 @@
 <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-<script src="https://d3js.org/d3.v4.min.js"></script>
 <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
 <script type='text/javascript'>
```
Comment (wynged) [inline home.html]: youre branch is still called foo

**Comment by wynged** [comment]
who should do the merge?  reviewer or other?

**Comment by MaxAbrahams** [comment]
Doesn't matter

**Comment by wynged** [comment]
don't forget to `git remote prune origin`

**Pull request merged**